worker-db-ddl.sql

```diff
@@ -38,26 +40,81 @@ CREATE TYPE cqc."WorkerDisability" AS ENUM (
 
 
 CREATE TABLE IF NOT EXISTS cqc."Worker" (
-	"ID" SERIALl NOT NULL PRIMARY KEY,
+	"ID" SERIAL NOT NULL PRIMARY KEY,
 	"WorkerUID" UUID NOT NULL,
 	"EstablishmentFK" INTEGER NOT NULL,
-	"NameOrID" VARCHAR(50) NOT NULL,
-	"Contract" cqc."WorkerContract" NOT NULL,
-	"MainJobFK" INTEGER NOT NULL,
+	"NameOrIdValue" VARCHAR(50) NOT NULL,
```

Comment:
Hi Warren, not fully aware of full background of this but i think this will only reflect last change not the full history over time as it would be always single record; is that ok? i thought we have separate table like workAudit and have those fields there. Any change will create a new record of last value and next value

Reply:
There is a separate `WorkerAudit` table, which includes a time history of all events.

The Worker main table keeps the most recent 'audit event' for each property. This makes it easier for someone to inspect/check manually (if using a direct connection to the DB), but also simplifies the queries, in that we can inspect the most recent event for each properly, without having to resolve (join) against the audit history and filter based on property name.